be/src/vec/sink/writer/vtablet_writer.h

```diff
@@ -114,6 +106,10 @@ struct AddBatchCounter {
     }
 };
 
+struct WriteBlockCallbackContext {
```

Comment:
Only one variable, no need this structure？

Reply:
introduced in progress of this pr. for more change possible in the future I think this is more friendly so didn't remove this.